composer.json

```
@@ -11,12 +11,13 @@
     }
   ],
   "require": {
-    "guzzlehttp/guzzle": "^7.0",
-    "php": "~8.1.0 || ~8.2.0 || ~8.3.0"
+    "guzzlehttp/guzzle": "^7.9.2",
+    "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^10.5",
-    "ergebnis/phpunit-slow-test-detector": "^2.15"
+  	"guzzlehttp/promises": ">=2.0.4",
```

Just to fix indentation.
```suggestion
    "guzzlehttp/promises": ">=2.0.4",
```